src/applications/claims-status/components/StemDeniedDetails.jsx

```diff
@@ -0,0 +1,194 @@
+        You met these criteria for the Rogers STEM Scholarship:
+      </h3>
+      <ul className="stem-ad-list">
+        <li className="stem-ad-list-item">
+          You are eligible for Post 9/11 GI Bill benefits
+          <ul className="stem-ad-list-secondary">
+            <li className="stem-ad-list-item">
+              By law, all Rogers STEM scholarship recipients must be eligible
+              for Post 9/11 GI Bill (Chapter 33) benefits. (38 U.S. Code §
+              3320).
+            </li>
+            <li className="stem-ad-list-item">
+              We reviewed your service and school data records currently on
+              file. As of [{date}
+              ], you are eligible for Post 9/11 GI Bill benefits.
+            </li>
+          </ul>
+        </li>
+        {(isEnrolledStem || isPursuingTeachingCert) && (
+          <li>
+            You meet the degree requirements for the Rogers STEM Scholarship
+            <ul className="stem-ad-list-secondary">
+              <li className="stem-ad-list-item">
+                By law, all Rogers STEM Scholarship recipients must either be:
+                <ol>
+                  <li>Enrolled in an eligible STEM undergraduate degree or</li>
+                  <li>
+                    Have an eligible STEM undergraduate degree and be pursuing a
+                    teaching degree
+                  </li>
+                </ol>
+              </li>
+              <li className="stem-ad-list-item">
+                You meet the degree requirements because you answered "Yes" to
+                one of these questions on the Rogers STEM Scholarship
+                application.
+                <ul className="stem-ad-list-secondary">
+                  <li className="stem-ad-list-item">
+                    "Are you enrolled in a science, technology, engineering, or
+                    math (STEM) degree program?" or
+                  </li>
+                  <li className="stem-ad-list-item">
+                    "Do you have a STEM undergraduate degree and are now
+                    pursuing a teaching certification?"
+                  </li>
+                </ul>
+              </li>
+            </ul>
+          </li>
+        )}
+      </ul>
+      <h3 className="vads-u-font-family--sans">
+        What you should do if you disagree with our decision
+      </h3>
+      <p>
+        If you disagree with our decision, you have 1 year from the date of this
+        letter to request a decision review or appeal. You have 3 options to do
+        this:
+      </p>
+      <ul className="stem-ad-list">
+        <li className="stem-ad-list-item">
+          If you have new or previously unprovided evidence to prove your
+          eligibility for this scholarship, you can file a Supplemental Claim by
+          completing VA Form 20-0995.{' '}
+          <a href="https://www.va.gov/find-forms/about-form-20-0995/">
+            Download VA Form 20-0995.
+          </a>
```

Comment:
This hasn't been formally documented, but a download link also has recommended markup. See https://github.com/department-of-veterans-affairs/vets-design-system-documentation/issues/152#issuecomment-553127340

Example:

```html
<a 
  href="https://www.va.gov/playbook/downloads/Voices_Of_Veterans.pdf" 
  download="Voices_Of_Veterans.pdf" 
  type="application/pdf">
    <i aria-hidden="true" class="fas fa-download vads-u-padding-right--1" role="img"></i>
      Download the Voices of Veterans <dfn><abbr title="Portable Document Format">PDF</abbr> (4.9<abbr title="Megabytes">MB</abbr>)</dfn>
</a>
```